package/Dockerfile

```diff
@@ -30,9 +32,8 @@ RUN apk -U --no-cache add curl wget ca-certificates tar sysstat acl\
     && apk del curl
 
 RUN mkdir -p /opt/cni/bin
-RUN wget -q -O - https://github.com/containernetworking/plugins/releases/download/${CNI_PLUGINS_VERSION}/cni-plugins-linux-${ARCH}-${CNI_PLUGINS_VERSION}.tgz | tar xzf - -C /tmp \
-    && wget -q -O /tmp/flannel "https://github.com/flannel-io/cni-plugin/releases/download/${FLANNEL_VERSION}/flannel-${ARCH}" \
-    && chmod +x /tmp/flannel
```

Comment:
Could you remove the env var CNI_PLUGINS_VERSION and FLANNEL_VERSION at lines 4 and 5?

Reply:
Inculcated the changes.